### DISCUSSION TITLE
Mismatch Trakt vs Plex/Sonarr

### DISCUSSION THREAD

**Discussion by glensc** [Taxel/PlexTraktSync#1309] [Q&A] [answered]
So there's some series which episodes are shifted by one position.

Sonarr and Plex agree on the titles, but Trakt does not.

What are the solutions here? What to check or change? or need PlexTraktSync to offer some mapping?

**Comment by glensc** [comment, marked as answer]
On the Plex side, seems guid is thetvdb: `PlexGuid:com.plexapp.agents.thetvdb://xxx/1`

**Reply by glensc** [reply]
Found that can switch agent under Fix Match for  the Series

<img width="345" alt="image" src="https://user-images.githubusercontent.com/199095/209140533-0d595eda-99ed-48b3-bdaf-5f5da3aee38a.png">

Under best practices it suggests to use tmdb:
- https://github.com/Taxel/PlexTraktSync/#good-practices

I assume it's The Movie Database.

it changed type to local, I guess it takes time for it to change while scanning

```
Media.Type: 'season'
Media.Guid: 'local://11772'
```

**Reply by glensc** [reply]
yep. now it's 

```
Media.Type: 'season'
Media.Guid: 'com.plexapp.agents.themoviedb://xxx/1?lang=en'
```

and matches trakt

**Reply by simonc56** [reply]
Yes, but now it'll mismatch Sonarr because Sonarr uses TVDB.
Hence the need to use the new Plex agent 😉

**Comment by simonc56** [comment]
> some series

Be specific. Give names and urls please.
Also provide logs if possible.

PS: old Plex agent is bad practice.

**Reply by glensc** [reply]
I did not disclose details due privacy concerns.

I use legacy agent because new agent is slower for series (needs to fetch episode details with separate API calls).

**Reply by simonc56** [reply]
> Under best practices it suggests to use tmdb:
>
>   [Taxel/PlexTraktSync#good-practices](https://github.com/Taxel/PlexTraktSync/#good-practices)
>
> I assume it's The Movie Database.

No need to assume, it's written in the section you quoted:
>Organize your shows folders and naming according to [Plex standard](https://support.plex.tv/articles/naming-and-organizing-your-tv-show-files/) and [theMovieDatabase](https://themoviedb.org/) (tmdb) order.
